Exdi/exdigdbsrvsample/GdbSrvControllerLib/XmlDataHelpers.h

```diff
@@ -94,20 +94,26 @@ namespace GdbSrvControllerLib
 
     //  Type describes the vector Register structure
     typedef std::vector<RegistersStruct> vectorRegister;

```

Comment:
Nit: I know this isn't new, but I think "vector register" is a slightly confusing name, since there are actually registers that are vectors. I'd probably call it a "RegisterVector". Same for the others... if you're going to have a typedef for mapGdbServerRegister, I think it's more common to call it GdbServerRegisterMap

Reply:
Thanks, make sense totally